overlord/state/taskrunner.go

```diff
@@ -69,9 +69,8 @@ func (r *TaskRunner) Handlers() map[string]HandlerFunc {
 	return r.handlers
 }
 
-// taskFail marks task t and all tasks waiting (directly and indirectly on it) as in ErrorStatus.
-func taskFail(task *Task) {
-	task.SetStatus(ErrorStatus)
+// propagateError sets to ErrorStatus all tasks directly and indirectly waiting on t.
```

Comment:
I've inverted the phrasing moving "to ErrorStatus" to the end. Hopefully that's more clear.